kubernetes/Dockerfile

```diff
@@ -40,19 +40,13 @@ RUN apt-get update \
     xfsprogs \
     && rm -rf /var/lib/apt/lists/*
 
-COPY --from=build /go/src/k8s.io/kubernetes/_output/bin/hyperkube /usr/local/kubernetes/bin/hyperkube
+COPY --from=build /go/src/k8s.io/kubernetes/_output/bin/kube-apiserver /usr/local/kubernetes/bin/kube-apiserver
+COPY --from=build /go/src/k8s.io/kubernetes/_output/bin/kube-controller-manager /usr/local/kubernetes/bin//kube-controller-manager
+COPY --from=build /go/src/k8s.io/kubernetes/_output/bin/kube-proxy /usr/local/kubernetes/bin/kube-proxy
+COPY --from=build /go/src/k8s.io/kubernetes/_output/bin/kube-scheduler /usr/local/kubernetes/bin/kube-scheduler
+COPY --from=build /go/src/k8s.io/kubernetes/_output/bin/kubectl /usr/local/kubernetes/bin/kubectl
+COPY --from=build /go/src/k8s.io/kubernetes/_output/bin/kubelet /usr/local/kubernetes/bin/kubelet
 COPY --from=build /go/src/k8s.io/kubernetes/LICENSE /usr/local/kubernetes/LICENSE
 COPY install-tools /usr/local/kubernetes/install-tools
```

Comment:
`install-tools` is not used any longer.  Please remove this line and the script itself.